### ISSUE TITLE
Applications installed via AM do not have icons. 

### ISSUE THREAD

**Issue by musicfunfan** [ivan-hc/AM#561]
Hello there. I use the AM package manager for some time now. And is working very good. The applications have a desktop entry on the application launcher (Rofi), but they do not have an icon. 
Is it possible to display the icon of the application? 

In this screenshot, i show the Joplin application installed via AM and not having the Joplin icon. 

![εικόνα](https://github.com/ivan-hc/AM/assets/86926443/f858608c-0f0c-4303-a269-3616d5877a88)

I did try the same thing on KDE 6 and its has icons. So this is probably my fault... 

Any idea on how to fix it ? 

I think i need to find where the icons has been stored. And create some symlink or ... add something on my PATH ? maybe ?

**Comment by musicfunfan** [comment]
I do not use regular Rofi. I use ``rofi-lbonn-wayland-git`` Because i am on Wayland and regular Rofi do not work well on Wayland. I use this fork for some time now and is fine. 

I have a Rofi emoji fork. Because i wanted to make a PR. But now i do not remember what i wanted to add... But it was something in the documentation side. \

I also use flatpaks,snaps,arch packages and Rofi display the icons just fine... I do not know why appimages does not... I was thinking because the icons is not on the PATH or something of this nature.

**Comment by Samueru-sama** [comment]
> I also use flatpaks,snaps,arch packages and Rofi display the icons just fine... I do not know why appimages does not... I was thinking because the icons is not on the PATH or something of this nature.

It's a bug on rofi that if the icon has a path and the image doesn't have .png or .svg on the extension, it fails to display the image. 

The bug was fixed once again, you would need to ask the fork you are using to apply [the fix](https://github.com/davatorium/rofi/commit/9b6e70b365116935158413034cf2aeb16affd9d4)

**Comment by ivan-hc** [comment]
I can't reproduce the issue

![Istantanea_2024-05-06_17-40-37 png](https://github.com/ivan-hc/AM/assets/88724353/10dc5e03-3350-441d-bf6c-8699af2fde3d)

maybe it's your menu missing some specs

**Comment by Samueru-sama** [comment]
> I can't reproduce the issue

The issue is when rofi is used as the application launcher, to do that run `rofi -show drun -show-icons`

However issue happens with the last stable release of rofi, which is version 1.7.5 which released in **2022**, it was recently fixed like a month ago after I reported the issue, which I linked in my first comment here. 

The rofi appimage is build by cloning the repo instead. So it has the fix and you will never see the issue.

If you want to reproduce the issue you would need to install rofi using your distros package manager and run rofi with `rofi -show drun -show-icons` and you will see that the apps installed by am have missing icons.
